content/en/blog/2021-04-27-cant-read-wont-buy.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Can't Read, Won't Buy"
+slug: "cant-read-wont-buy"
+date: "2021-04-27T11:24:40-04:00"
+draft: false
```

Comment:
Perhaps it would be easiest to push blog posts as `draft: false` when they're ready for review? That will allow one-click publishing and hopefully make things a bit easier.

Reply:
got it, will do next time